invitations/tests/allauth/test_allauth.py

```diff
@@ -65,6 +65,38 @@ def test_accept_invite_allauth(self, method):
             email='email@example.com')
         self.assertTrue(allauth_email_obj.verified)
 
+    @parameterized.expand([
+        ('get'),
+        ('post'),
```

Comment:
If these are meant to be tuples, they’re missing trailing commas. (If not, remove parentheses.)